agent/util-scripts/pbench-tool-meister-start

```diff
@@ -340,6 +341,12 @@ def main(argv):
         ), f"bad channel: {resp!r}"
         assert resp["data"] == 1, f"bad data: {resp!r}"
 
+    # 2.5. Add tool metadata json to redis
+    tm_start_path = pathlib.Path(__file__).parent.absolute()
```

Comment:
Consider adding a `try/except` block here to capture errors from opening the file, reading from the file, and loading the JSON data.

Also, it seems `absolute` is not documented, can you use `resolve(strict=True)` instead?